tests/integration/components/select-box-test.js

```diff
@@ -32,7 +32,7 @@ describe('Integration | Component | select box', function () {
         this.set('valueKey', 'value');
         this.set('options', [{value:'1', display: 'first'}, {value:'2', display: 'second'}, {value:'3', display: 'third'}, {value:'4', display: 'fourth'}]);
         this.render(hbs `{{select-box options=options displayKey=displayKey valueKey=valueKey nativeSelect=true}}`);
-        expect(findAll('[data-test-id^="sbNativeDeepOption"]')).to.have.lengthOf(4);
```

Comment:
I'm curious if there is still a way we can use test ids. Using the elements to select in the test seems fine though.

Reply:
Does `select-light` have a className assignment? If so, we can use that if keeping the test id is not an option.  I'd like for test selectors to be more specific than element names.